src/util.ts

```diff
@@ -348,7 +348,7 @@ export function getToolsGopath(useCache: boolean = true): string {
 
 function resolveToolsGopath(): string {
```

Comment:
In `goLint.ts` the toolsGopath is accessed directly from the config, can you update that to use `resolveToolsGopath` instead?